Test/regressions/InternalFunctionCall_fail.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity >=0.4.24 <0.6.0;
```

Comment:
Can we add this file to the regression, just to make sure we keep this file up to date.